PluralKit.Bot/Proxy/ProxyService.cs

```diff
@@ -188,6 +188,56 @@ public bool ShouldProxy(Channel channel, Message msg, MessageContext ctx)
         await HandleProxyExecutedActions(ctx, autoproxySettings, trigger, proxyMessage, match);
     }
 
+    public async Task ExecuteReproxy(Message trigger, PKMessage msg, ProxyMember member)
+    {
+        var originalMsg = await _rest.GetMessageOrNull(msg.Channel, msg.Mid);
+        if (originalMsg == null)
+            throw new PKError("Could not reproxy message.");
+
+        // Get a MessageContext for the original message
+        MessageContext ctx =
+            await _repo.GetMessageContext(msg.Sender, msg.Guild!.Value, msg.Channel);
+
+        var autoproxySettings = await _repo.GetAutoproxySettings(ctx.SystemId.Value, msg.Guild!.Value, null);
```

Comment:
this seems unused?

Reply:
autoproxySettings is only here because it needs to be passed to HandleProxyExecutedActions, we don't actually need it otherwise